Mathlib/Data/Polynomial/Basic.lean

```diff
@@ -727,6 +727,9 @@ theorem coeff_C_zero : coeff (C a) 0 = a :=
 theorem coeff_C_ne_zero (h : n ≠ 0) : (C a).coeff n = 0 := by rw [coeff_C, if_neg h]
 #align polynomial.coeff_C_ne_zero Polynomial.coeff_C_ne_zero
 
+theorem coeff_nat_cast_ite : (Nat.cast m : R[X]).coeff n = ite (n = 0) m 0 := by
```

Comment:
Should this be marked `simp`? I'm not sure and I'll defer to your judgment.

Reply:
Yes, you are probably right.  There isn't a lot of consistency in labeling as `simp` in `Polynomial`, but there is the precedent of docs#Polynomial.natDegree_monomial.

Reply:
@j-loreaux I went ahead with your `simp` suggestion and Lean found a golfing opportunity!  Let's see if the PR builds again.